Modules/Tracker/QuestieTrackerPrivates.lua

```diff
@@ -8,56 +8,335 @@ local QuestieCombatQueue = QuestieLoader:ImportModule("QuestieCombatQueue")
+                    xOff = xOn + width
+                    yOff = yOn
+                elseif preSetPoint == "BOTTOMLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn + height
+                elseif preSetPoint == "BOTTOMRIGHT" then
+                    xOff = xOn
+                    yOff = yOn + height
+                end
+                return xOff, yOff
+
+            -- To "BOTTOMLEFT" from...
+            elseif setPoint == "BOTTOMLEFT" then
+                if preSetPoint == "TOPRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn + -height
+                elseif preSetPoint == "TOPLEFT" then
+                    xOff = xOn
+                    yOff = yOn + -height
+                elseif preSetPoint == "BOTTOMRIGHT" then
+                    xOff = xOn + -width
+                    yOff = yOn
+                end
+                return xOff, yOff
+
+            -- To "BOTTOMRIGHT" from...
+            elseif setPoint == "BOTTOMRIGHT" then
+                if preSetPoint == "TOPRIGHT" then
+                    xOff = xOn
+                    yOff = yOn + -height
+                elseif preSetPoint == "TOPLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn + -height
+                elseif preSetPoint == "BOTTOMLEFT" then
+                    xOff = xOn + width
+                    yOff = yOn
+                end
+                return xOff, yOff
+            end
+        else
+            return xOn, yOn
+        end
+    end
+end
+
+function _QuestieTracker:OnResizeStart(button)
+    Questie:Debug(DEBUG_DEVELOP, "[_QuestieTracker:OnResizeStart]", button)
+    if InCombatLockdown() then return end
+    local baseFrame = QuestieTracker:GetBaseFrame()
+
+    if button == "LeftButton" then
+        if IsMouseButtonDown(button) then
+            if IsControlKeyDown() or not Questie.db.global.trackerLocked then
+                _QuestieTracker.isSizing = true
+                tempTrackerLocation = {baseFrame:GetPoint()}
+                baseFrame:StartSizing("RIGHT")
+                _QuestieUpdateTimer = C_Timer.NewTicker(0.1, function()
+                    local baseFrame = QuestieTracker:GetBaseFrame()
+                    Questie.db.char.TrackerWidth = baseFrame:GetWidth()
+                    QuestieTracker:ResetLinesForChange()
+                    QuestieTracker:Update()
+                end)
+            end
+        end
+    elseif button =="RightButton" then
+        Questie.db.char.TrackerWidth = 0
+        QuestieTracker:ResetLinesForChange()
+        QuestieTracker:Update()
+        _QuestieTracker.baseFrame.sizer:SetAlpha(1)
+    end
+end
+
+function _QuestieTracker:OnResizeStop(button)
+    Questie:Debug(DEBUG_DEVELOP, "[_QuestieTracker:OnResizeStop]", button)
+    local baseFrame = QuestieTracker:GetBaseFrame()
+    if button == "RightButton" or _QuestieTracker.isSizing ~= true then return end
```

Comment:
I hate when a return is so far too the right. So easy to oversee this...

```suggestion
    if button == "RightButton" or _QuestieTracker.isSizing ~= true then
        return
    end
```

Reply:
True... I hate vertical white space. Squashing such a short statement into a "one liner" is visually more appealing. But, I'll stick to your style suggestions. :)